virt/virt-2-4-release-notes.adoc

```diff
@@ -147,8 +147,22 @@ $ oc delete pod -n openshift-cnv --selector=kubevirt.io=virt-handler --field-sel
 ----
 <1> Where <source-node-name> is the name of the source node that the VMI is migrating from.
 
-//https://bugzilla.redhat.com/show_bug.cig?=id=1959237
-* If you upgrade to {VirtProductName} {VirtVersion}, both older and newer versions of common templates are available for each combination of operating system, workload, and flavor. When you create a virtual machine by using a common template, you must use the newer version of the template. Disregard the older version to avoid issues. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1859237[*BZ#1859237*])
+* Common templates in previous versions of {VirtProductName} had a default `spec.terminationGracePeriodSeconds` value of `0`. Virtual machines created from these older common templates can encounter disk issues from being forcefully terminated. +
+If you upgrade to {VirtProductName} {VirtVersion}, both older and newer versions of common templates are available for each combination of operating system, workload, and flavor. When you create a virtual machine by using a common template, you must use the newer version of the template. Disregard the older version to avoid issues. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1859235[*BZ#1859235*])
+
+** To verify if a virtual machine is impacted, run the following command in the namespace of the virtual machine to determine the `spec.terminationGracePeriodSeconds` value:
```

Comment:
My one nit is that "To verify if a virtual machine is impacted" strikes me as potentially weird phrasing for non-native English speakers. I would change it to "To determine if you need to patch a specific virtual machine," or "To verify if a virtual machine is affected by this bug," or something like that.